packages/engine-server/lib/engineConnector.js

```diff
@@ -17,11 +17,19 @@ const bookings = amqp
   /* .subscribe is supposed to default to {noAck: true}, dont know what
    * it means but messages are not acked if i don't specify this
    */
-  .subscribe({ noAck: true }, [
-    routingKeys.NEW,
-    routingKeys.ASSIGNED,
-    routingKeys.DELIVERED,
-  ])
+  .subscribe({ noAck: true }, [routingKeys.ASSIGNED, routingKeys.DELIVERED])
+  .map((bookings) => {
+    return { ...bookings.json(), status: bookings.fields.routingKey }
+  })
+
+const bookingsNewWithRoutes = amqp
```

Comment:
Can this be unassignedBookings instead? or newBookings? :) It's not a super big deal but perhaps important to keep naming as clear as possible.